librz/core/cmd_descs/cmd_meta.yaml

```
@@ -0,0 +1,407 @@
+# SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
+# SPDX-License-Identifier: LGPL-3.0-only
+---
+name: cmd_meta
+commands:
+  - name: C
+    cname: meta_list
+    summary: List all meta information
+    type: RZ_CMD_DESC_TYPE_ARGV_STATE
+    modes:
+      - RZ_OUTPUT_MODE_STANDARD
+      - RZ_OUTPUT_MODE_RIZIN
+      - RZ_OUTPUT_MODE_JSON
+      - RZ_OUTPUT_MODE_LONG
+    args: []
+  - name: C.
+    cname: meta_list_at
+    summary: Show all meta information at current address
+    type: RZ_CMD_DESC_TYPE_ARGV_STATE
+    modes:
+      - RZ_OUTPUT_MODE_STANDARD
+      - RZ_OUTPUT_MODE_RIZIN
+      - RZ_OUTPUT_MODE_JSON
+      - RZ_OUTPUT_MODE_LONG
+    args: []
+  - name: C-
+    cname: meta_remove
+    summary: Remove meta information at current address
+    args: []
+  - name: C-*
+    cname: meta_remove_all
+    summary: Remove all meta information
+    args: []
+  - name: CC
+    summary: Manipulate the comments
+    subcommands:
+      - name: CC
+        cname: comment
+        summary: Append comment / List all comments
+        type: RZ_CMD_DESC_TYPE_ARGV_STATE
+        modes:
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_RIZIN
+          - RZ_OUTPUT_MODE_JSON
+          - RZ_OUTPUT_MODE_LONG
+        args:
+          - name: text
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: CC.
+        cname: comment_at
+        summary: Show comment at the current offset
+        args: []
+      - name: CC+
+        cname: comment_append
+        summary: Append comment at current address
+        args:
+          - name: text
+            type: RZ_CMD_ARG_TYPE_STRING
+      - name: CC-
+        cname: comment_remove
+        summary: Remove comment at current address
+        args: []
+      - name: CC-*
+        cname: comment_remove_all
+        summary: Remove all comments
+        args: []
+      - name: CC,
+        cname: comment_filelink
+        summary: Show / Set comment file
+        args:
+          - name: file
+            type: RZ_CMD_ARG_TYPE_FILE
+            optional: true
+      - name: CCa
+        cname: comment_add_at
+        summary: Add comment at the specified address
+        args:
+          - name: address
+            type: RZ_CMD_ARG_TYPE_RZNUM
+          - name: text
+            type: RZ_CMD_ARG_TYPE_STRING
+      - name: CCe
+        cname: comment_editor
+        summary: Edit comment using `cfg.editor`
+        args: []
+      - name: CCf
+        summary: List comments in function at current address
+        subcommands:
+          - name: CCf
+            cname: comment_function_list
+            summary: Append comment / List all comments
```

```suggestion
            summary: List all comments
```

I guess this only lists now, since this command has no arguments, right?